lab05-2.cpp

```diff
@@ -143,10 +143,10 @@ vector<string> canonicalize(const std::string p)
 /************************************
  * Compare paths and output results *
  ************************************/
-void runHomographFunction(const string path1, const string path2)
+void runHomographFunction(const string path1, const string path2, const string base_path = getBasePath())

```

Comment:
an option to pass in our own base_path to use for test cases. It defaults to the user's actual base path